### PR TITLE
PTV-1753: leftovers

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

A couple of leftovers from the PTV-1753 ticket:
- Switch the log and results tab over to using the job manager
- Delete the old Job State Viewer module (no longer needed)

ETA: backend tests have failed; frontend tests have miraculously all passed!

# Jira Ticket / Issue #

Related Jira ticket: https://kbase-jira.atlassian.net/browse/PTV-1753
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative, creating an app cell, and running it through the various stages.

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/static/kbase/js/common/jobStateViewer.js [hunk 1]
```diff
@@ -101,9 +101,11 @@ define([
                 this.lastJobState = this.jobManager.getJob(this.jobId);
                 this.renderJobState(this.lastJobState || { job_id: this.jobId });
 
-                this.bus.emit(jcm.MESSAGE_TYPE.STATUS, {
-                    [jcm.PARAM.JOB_ID]: this.jobId,
-                });
+                if (!this.lastJobState || !this.lastJobState.status) {
+                    this.bus.emit(jcm.MESSAGE_TYPE.STATUS, {
+                        [jcm.PARAM.JOB_ID]: this.jobId,
+                    });
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/jobStateViewer.js:107]: only request job status if there isn't a complete job state object (although that should all be handled by the job manager)

File: kbase-extension/static/kbase/js/common/simpleLogViewer.js [hunk 2]
```diff
@@ -710,11 +710,10 @@ define([
                         // some kind of 'await job info' message
                         this.ui.setContent(LOG_PANEL, p(this.messages.JOB_STATUS_UNKNOWN));
                         this._renderButtonState('default');
+                        this.bus.emit(jcm.MESSAGE_TYPE.STATUS, {
+                            [jcm.PARAM.JOB_ID]: this.jobId,
+                        });
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/simpleLogViewer.js:715]: same as above

File: nbextensions/appCell2/widgets/tabs/logTab.js [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
+    'bluebird',
+    'common/html',
+    'common/ui',
+    'common/jobStateViewer',
+    'common/simpleLogViewer',
+], (Promise, html, UI, JobStateViewerModule, SimpleLogViewerModule) => {
     'use strict';
 
-    const { JobLogViewer } = JobLogViewerModule;
+    const { JobStateViewer } = JobStateViewerModule;
```
Comment (ialarmedalien) [inline nbextensions/appCell2/widgets/tabs/logTab.js:10]: use the new job state and job log viewer modules as we now have the job manager in charge of all things job.

File: test/unit/spec/common/simpleLogViewer-spec.js [hunk 4]
```diff
@@ -359,9 +359,10 @@ define([
                             this.simpleLogViewerInstance.messages.BATCH_JOB
                         );
                         expect(this.bus.emit.calls.allArgs()).toEqual(
-                            jasmine.arrayWithExactContents([
-                                [jcm.MESSAGE_TYPE.STATUS, { [jcm.PARAM.JOB_ID]: jobId }],
```
Comment (ialarmedalien) [inline test/unit/spec/common/simpleLogViewer-spec.js:363]: no longer send a job status request at start up -- that's handled by the job manager

File: test/unit/spec/common/simpleLogViewer-spec.js [hunk 5]
```diff
@@ -951,9 +951,6 @@ define([
                         input = generateInput(jobId);
                     let acc = -1;
 
-                    // // set the job manager's status request loop going
-                    this.jobManager.restoreFromSaved();
-
```
Comment (ialarmedalien) [inline test/unit/spec/common/simpleLogViewer-spec.js:956]: move this later in the function so now we capture the first message (which requests a status update for the whole batch)

**Review by briehl — APPROVED** [review]
LGTM!

**Pull request merged**